pkg/env/base_image.go

```diff
@@ -4,7 +4,8 @@ import "time"
 
 var (
 	// BaseImageWatcherEnabled controls whether the base image watcher is enabled.
-	BaseImageWatcherEnabled = RegisterBooleanSetting("ROX_BASE_IMAGE_WATCHER_ENABLED", false)
+	// This setting is only consulted when the ROX_BASE_IMAGE_DETECTION feature flag is true.
+	BaseImageWatcherEnabled = RegisterBooleanSetting("ROX_BASE_IMAGE_WATCHER_ENABLED", true)
```

Comment:
Should we make a feature flag for base image feature? `BaseImageWatcherEnabled` seems a bit odd to me because it makes it possible to turn on the base image feature without running base image watcher (if base image watcher is switched off)? Let me know if I am misunderstanding something.